ZeldaFullEditor/Data/OverworldMap.cs

```diff
@@ -324,7 +325,38 @@ public OverworldMap(byte index, Overworld overworld)
                 }
 
                 // Set the mosaic values.
-                this.Mosaic = index == 0x00 || index == 0x40 || index == 0x80 || index == 0x81 || index == 0x88;
+                switch (index)
```

Comment:
these could probably use constants or at least some labeling of the significance of these, e.g. lost woods/skull woods vanilla mosaic effects

Reply:
Done